biogeochem/EDCanopyStructureMod.F90

```diff
@@ -1784,7 +1784,7 @@ subroutine update_hlm_dynamics(nsites,sites,fcolumn,bc_out)
 
     use EDTypesMod        , only : ed_patch_type, ed_cohort_type, &
          ed_site_type, AREA
-    use FatesInterfaceTypesMod , only : bc_out_type
+    use FatesInterfaceTypesMod , only : bc_in_type, bc_out_type
```

Comment:
this new declaration of bc_in_type can be removed I think

Reply:
Will remove.